packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -1611,9 +1431,7 @@ describe( 'Selecting links', () => {
 				// Only entity searches contain more than 1 suggestion.
 				if ( type === 'entity' ) {
 					// Check we can go down again using the down arrow.
-					act( () => {
-						Simulate.keyDown( searchInput, { keyCode: DOWN } );
-					} );
+					await triggerArrowDown( searchInput );
 
 					selectedSearchResultElement = container.querySelector(
 						'[role="option"][aria-selected="true"]'
```

Comment:
Should be replaceable with:

```suggestion
					selectedSearchResultElement = screen.queryByRole(
						'option',
						{ selected: true }
```